src/contracts/Event/NameSchema/ResolveUrlAliasSchemaEvent.php

```diff
@@ -0,0 +1,29 @@
+<?php
+
+/**
+ * @copyright Copyright (C) Ibexa AS. All rights reserved.
+ * @license For full copyright and license information view LICENSE file distributed with this source code.
+ */
+declare(strict_types=1);
+
+namespace Ibexa\Contracts\Core\Event\NameSchema;
+
+use Ibexa\Contracts\Core\Repository\Values\Content\Content;
+
+final class ResolveUrlAliasSchemaEvent extends AbstractSchemaEvent implements ContentAwareEventInterface
+{
+    protected Content $content;
```

Comment:
You may consider introducing `ContentAwareEventTrait` for these.

```suggestion
    private Content $content;
```